supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -317,6 +317,17 @@ The API object for a deployment can be either a Kubernetes-native `Deployment` o
 
 *See also*:
 
+[discrete]
+[[developer-preview]]
+==== image:images/yes.png[yes] Developer Preview (noun)
+*Description*: Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. The software might not have any documentation, is subject to change or removal at any time, and testing is limited. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready.
```

Comment:
```suggestion
*Description*: Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. The software might not have any documentation, is subject to change or removal at any time, and testing is limited. Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready.
```
- Not sure, but are we supposed to be adding `{nbsp}` between `Red` and `Hat` now?

Reply:
I wouldn't worry about {nbsp} for now. We have issue #274 open to do a sweep through the guide to add as necessary.

As for the wording, some of it is used that way in the TP section too. So I might leave the first part the same, but the second part (adding "of it") isn't anywhere else, so you could potentially do that part.

Reply:
Nice work, @aireilly . 

I've some minor suggestions:

- Italize the first instance of _Developer Preview_. 
- Developer Preview software is not supported by Red Hat in any way and is not functionally complete or production-ready. > Red Hat does not support Developer Preview software, because the software is not functionally complete or production-ready.
- The software might not have any documentation, is subject to change or removal at any time, and has received limited testing. > The software might not include documentation, could undergo software changes or removal at any time, and might receive limited testing.